scss/_colors.scss

```diff
@@ -26,5 +26,6 @@ $lightbox: rgba(0, 0, 0, 0.5);
 $extra-dark-gray: #2d2d2d;
 $dark-gray: #888e8e;
 $medium-gray: #cacbcc;
+$very-light-gray: #dedede;
```

Comment:
Can't we define a standard way to generate names for these color scales?
This way of naming them doesn't seems scalable if we need for example something between medium-gray and dark-gray...
Also it isn't intuitive as we have a very-light-gray and extra-dark-gray, both have the same idea but with a different prefix... the medium-gray is quite light so it seems strange to have the prefix medium...

Reply:
I think that's a valid discussion. The idea is to use the same name in both the codebase and in the design team's Zeplin projects. So if we want to define a naming convention, a conversation should be arranged with both teams.